modules/040-node-manager/images/bashible-apiserver/pkg/cmd/server/start.go

```diff
@@ -52,6 +52,22 @@ func NewBashibleServerOptions(out, errOut io.Writer) *BashibleServerOptions {
 		StdErr: errOut,
 	}
 	o.RecommendedOptions.Etcd = nil
+	o.RecommendedOptions.SecureServing.CipherSuites = []string{
```

Comment:
Why do yo build static list instead of using CipherSuites() func ?

Reply:
If we use `tls.CipherSuites()` then the http2 library returns an error `main.go:23 error configuring http2: http2: TLSConfig.CipherSuites index 5 contains an HTTP/2-approved cipher suite (0x1301), but it comes after unapproved cipher suites. With this configuration, clients that don't support previous, approved cipher suites may be given an unapproved one and reject the connection.`